quinn-proto/src/config.rs

```diff
@@ -316,6 +338,67 @@ impl fmt::Debug for TransportConfig {
     }
 }
 
+/// Parameters governing PLPMTUD.
+///
+/// PLPMTUD performs a binary search, trying to find the highest packet size that is still
+/// supported by the current network path. The lower bound of the search is equal to
+/// [`TransportConfig::initial_max_udp_payload_size`] in the initial PLPMTUD round, and is equal
+/// to the currently discovered MTU in subsequent runs. The upper bound is determined by
+/// [`PlpmtudConfig::max_udp_payload_size_upper_bound`].
+///
+/// If, at some point, the network path no longer accepts packets of the detected size, packet
+/// loss will eventually trigger black hole detection and reset the detected MTU to 1200. In
+/// that case, PLPMTUD will be triggered after [`PlpmtudConfig::black_hole_cooldown`] (ignoring the
+/// timer that was set based on [`PlpmtudConfig::interval`]).
+#[derive(Clone, Debug)]
```

Comment:
Looks like we can make this `Copy`?

Reply:
We could, but that would cause the future addition of non-`Copy` fields to be breaking. Our other config structs aren't `Copy`, so I think it makes sense to be conservative here.

Reply:
Fair enough. Let's make all the private types that can be `Copy` `Copy`, though.

Reply:
Sure, just pushed a change for that